tests/testthat/test-chunks.R

```diff
@@ -40,3 +40,78 @@ test_that("produce informative errors for bad chunk_vars", {
   }, paste("chunk_vars must be a character vector;",
            "use chunk_vars=character() to specify 1 chunk"), fixed=TRUE)
 })
+
+data(breakpoints)
+only.error <- subset(breakpoints$error,type=="E")
+only.segments <- subset(only.error,bases.per.probe==bases.per.probe[1])
+signal.colors <- c(estimate="#0adb0a",
+                   latent="#0098ef")
+breakpointError <- 
+  list(signal=ggplot()+
+         geom_point(aes(position, signal, showSelected=bases.per.probe),
+                    data=breakpoints$signals)+
+         geom_line(aes(position, signal), colour=signal.colors[["latent"]],
+                   data=breakpoints$imprecision)+
+         geom_segment(aes(first.base, mean, xend=last.base, yend=mean,
+                          showSelected=segments,
+                          showSelected2=bases.per.probe),
+                      colour=signal.colors[["estimate"]],
+                      data=breakpoints$segments)+
+         geom_vline(aes(xintercept=base,
+                        showSelected=segments,
+                        showSelected2=bases.per.probe),
+                    colour=signal.colors[["estimate"]],
+                    linetype="dashed",
+                    data=breakpoints$breaks),
+       error=ggplot()+
+         geom_vline(aes(xintercept=segments, clickSelects=segments),
+                    data=only.segments, lwd=17, alpha=1/2)+
+         geom_line(aes(segments, error, group=bases.per.probe,
+                       clickSelects=bases.per.probe),
+                   data=only.error, lwd=4))
+
+bytes.used <- function(file.vec, ...){
+  ## Note: the --apparent-size flag gives sizes that are consistent
+  ## with file.info, but those sizes actually under-estimate the
+  ## actual amount of space used on disk.
+  file.str <- paste(file.vec, collapse=" ")
+  cmd <- paste("du --block-size=1", ..., file.str)
```

Comment:
go ahead and change the du code so that it works on your computer, or just comment it out.